src/Controls/tests/DeviceTests/Elements/Frame/FrameTests.cs

```diff
@@ -432,12 +433,20 @@ public async Task FramesWithinFrames(double widthConstraint, double heightConstr
 		async Task<Rect> LayoutFrame(Layout layout, Frame frame, double widthConstraint, double heightConstraint, Func<Task> additionalTests = null)
 		{
 			additionalTests ??= () => Task.CompletedTask;
+
+			// Let the OS call measure and arrange
+			var grid = new Grid()
+			{
+				Children = { layout }
+			};
+
 			return await
-					AttachAndRun(layout, async (handler) =>
+					AttachAndRun(grid, async (handler) =>
```

Comment:
Attach and run adds it to a grid already. And also, the exception happens when it is cleaning up the tests. It is removing the view from said Grid for the test.

Aaaaand, you saw my PR too: https://github.com/dotnet/maui/pull/19352